src/components/NotificationsController/NotificationsController.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { View } from 'react-native';
+
+import { Text, Button } from '..';
+import { removeNotification } from '../../store/notifications';
+
+const NotificationsController = () => {
+  const dispatch = useDispatch();
+  const { notifications } = useSelector(state => ({
+    notifications: state.notifications,
+  }));
+
+  if (!notifications) {
+    return null;
+  }
+
+  return (
+    <>
+      {notifications &&
+        notifications.map((notification, i) => (
+          <View key={i}>
```

Comment:
Probably better to use `notification.id` as a key, since we can close notifications in any order, react will have a hard time reordering them if you use index